client/apps/edit/components/layout/index.coffee

```diff
@@ -73,8 +64,8 @@ module.exports = class EditLayout extends Backbone.View
     }
 
   toggleAstericks: =>
-    @$('.edit-required + :input').each (i, el) =>
-      $(el).prev('.edit-required').attr 'data-hidden', !!$(el).val()
+    @$('.edit-required ~ :input').each (i, el) =>
+      $(el).prevAll('.edit-required').attr 'data-hidden', !!$(el).val()
```

Comment:
I don't see it being a problem. If it causes issues down the line we can be more explicit about targeting these asterisks rather than the magical searching nearby that this does.